streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -489,7 +490,11 @@ private void commitAll() {
     private void commitOne(AbstractTask task, long now) {
         try {
             task.commit();
+        } catch (CommitFailedException e) {
```

Comment:
Alright. I can change the comment. BTW, ```CommitFailedException``` is always created with a message text, "_Commit cannot be completed due to group rebalance_", in ```ConsumerCoordinator```. This is the only place I know that this exception is created.